website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -2893,6 +2893,183 @@ No response body.
 
 Status code `204`.
 
+## Show data retention policy
```

Comment:
Non-Blocking Nitpick (Same as above!): Should we titlecase this for consistency between the other headers and this one?:
```suggestion
## Show Data Retention Policy
```